pkg/apb/unbind.go

```diff
@@ -10,15 +10,15 @@ import (
 // github.com/op/go-logging, which is used all over the broker
 // Maybe apb defines its own interface and accepts that optionally
 // Little looser, but still not great
-func Unbind(instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger) error {
+func Unbind(instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger, parameters *Parameters) error {
```

Comment:
It would be nice to use the same order as `Bind`:

```
func Bind(
    instance *ServiceInstance,
    parameters *Parameters,
    clusterConfig ClusterConfig,
    log *logging.Logger,
) (string, *ExtractedCredentials, error) {
```